pkgs/development/libraries/vigra/default.nix

```
@@ -7,11 +7,10 @@
   fftw,
   fftwSinglePrec,
   hdf5,
-  ilmbase,
   libjpeg,
   libpng,
   libtiff,
-  openexr,
+  openexr_3,
```

```suggestion
  openexr,
```

If I understand the README of the Name-Based Package Directory correctly, we should specify dependency versions in `all_packages.nix` instead of hard-coding it in the override interface.

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults